main.go

```diff
@@ -118,11 +154,13 @@ func main() {
 		return handlers.LoggingHandler(log.Writer(), next)
 	})
 
-	// For primary nodes
+	router.HandleFunc("/", hcs.apiGetIsPrimary).Queries("max_allowable_byte_lag", "{max_allowable_byte_lag}").Methods("GET")
```

Comment:
Do we need Queries if we want the param to be optional?

Reply:
I also have `router.HandleFunc("/", hcs.apiGetIsPrimary).Methods("GET")` below so that is optional. Or is that not what you mean?

Reply:
Ah, I got it. `Queries` are not needed. Removed them.